pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -299,6 +305,8 @@ var _ = Describe("Pod Network", func() {
 
 			api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
 
+			mockNetwork.EXPECT().LinkByName(podInterface).Return(dummy, nil)
```

Comment:
Duplicates line 310

Reply:
I see, can you please use mockNetwork.EXPECT().LinkByName(podInterface).Return(dummy, nil).Times(2) instead of duplicating the line?